### ISSUE TITLE
Anderson bio baler not recognized as a baler

### ISSUE THREAD

**Issue by StefanBraun2001** [Courseplay/courseplay#3480]
Hello!

I downloaded the new anderson dlc. almost everthing works fine if you set the settings correctly. The only one that does not work is the bio baler. I belive, the problem is, that that thing is not recognized as a baler. When it runs full he says: Tool needs to be unloaded. Sometimes, when he stops to late, and it runs completly full (for usal, it stops at 99 % filling) he unload bales, but he does not stop.

**Comment by iljarogoff** [comment]
Tried this tool too, same as poster above. Additionally, in case the baler loads to 100% it unloads as it should, but it closes and instantly moves on, without waiting for the baler to close completely. So several meters of poplar are untouched

**Comment by PirateAE** [comment]
Any ETA on this getting addressed? kinda annoying to have to "assist" this for 6 hours on my feild...

**Comment by pvaiko** [comment]
If you post a savegame on a standard map with all equipment set up that'll sure make it happen faster as I don't have to spend time to set everything up :)

**Comment by Tensuko** [comment]
@pvaiko in case you would want to try to get it work:
[savegame5.zip](https://github.com/Courseplay/courseplay/files/5392666/savegame5.zip)
[MapDE.zip](https://github.com/Courseplay/courseplay/files/5392669/MapDE.zip)

Also contains a prepared Field + Tools for #3674

**Comment by d0x7** [comment]
Is there any update on this topic? 
I have the same issue that CP instantly starts driving again once the bale has been unloaded, therefore leaving some poplar untouched because it's not fully closed yet. I also get a notification every single bale that it needs to be unloaded (it's unloading automatically tho). Just now while I was writing this down, it got "stuck" at 2000 Liters, still showing 99%. I guess it tried to unload, but it couldn't be unloaded as it wasn't 100%. I disabled CP, drove until it was 100% and unloaded manually. Not sure whether this is an issue with balers in general, the bio baler itself or CoursePlay, but I've not seen this myself doing bales manually.

Another issue I just had: While switching lanes, the driver got kinda stuck in a "reverse driving loop", [here's a video](https://youtu.be/VBdOj0WCQLk) to see what I mean.

I know it's not the latest CoursePlay version (I'll update right after this, but I don't think this got fixed since there's no update on this issue). The settings regarding AI turn maneuvers, using pathfinder or that the pathfinder is allowed to reverse are all default (disabled). It just happend again, this time I enabled all debug channels and showed the log. It said something about colliding objects (I suppose the BioBaler with the tractor), hence I tried to move the BioBaler by hand, two times nothing happend and then it caught on the normal path again, very very strange. Does nobody else have these issues or are they just not reporting it? :/

[Second video](https://youtu.be/yEBwnpBR1k4) and the [corresponding log](https://gist.github.com/d0x7/b22166ac7cb2ae07977ba0de3836aee9).

Thank you :)
